seatunnel-core/seatunnel-core-starter/src/main/java/org/apache/seatunnel/core/starter/command/AbstractCommandArgs.java

```diff
@@ -41,6 +42,10 @@ public abstract class AbstractCommandArgs implements CommandArgs {
             description = "check config")
     private boolean checkConfig = false;
 
+    @Parameter(names = {"-n", "--name"},
```

Comment:
Please add the new parameter to the document https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/command/usage.mdx

Reply:
ok

Reply:
@EricJoy2048 Currently, this parameter is only supported for V2. Is there a special document for V2

Reply:
> 

Can we add V2 starter in this doc https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/command/usage.mdx ?

Reply:
> > 
> 
> Can we add V2 starter in this doc https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/command/usage.mdx ? @CalvinKirs

Reply:
in fact, `connectot-v2` it's not much different from v1, it's just the scripts and jars that are started. Are you willing to submit documents in this regard?

Reply:
> > 
> 
> Can we add V2 starter in this doc https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/command/usage.mdx ?

If it's like EricJoy2048 says, I can try

Reply:
if you have any question, you can ping me